artifactory/utils/filetree.go

```diff
@@ -0,0 +1,99 @@
+package utils
+
+import (
+	"strings"
+)
+
+var maxFilesInTree = 200
+
+type FileTree struct {
```

Comment:
I suggest adding the following description to this struct:

FileTree is a UI components that displays a file-system tree view in the terminal.